src/pages/Landing.tsx

```diff
@@ -35,11 +36,11 @@ export const Landing = () => {
   const renderModal = () => {
     switch (activeModal) {
       case ModalType.SIGNIN_SECRET_KEY:
-        return <SigninSecretKeyForm onClose={closeModal} />;
+        return <SignInSecretKeyForm onClose={closeModal} />;
       case ModalType.SIGNIN_TREZOR:
-        return <SigninTrezorForm onClose={closeModal} />;
+        return <SignInTrezorForm onClose={closeModal} />;
       case ModalType.SIGNIN_LEDGER:
-        return <div>Ledger</div>;
+        return <SignInLedgerForm />;
```

Comment:
Was it intentional to omit `onClose`?

Reply:
Ah ya on the current design/wireframe there isn't a cancel button for signing in with Ledger